test/unit/lodash.spec.js

```diff
@@ -0,0 +1,690 @@
+                    '4': [16],
+                    '5': [25],
+                    '9': [81],
+                    '11': [121],
+                    '1.4142135623730951': [2]
+                });
+            });
+        });
+        describe('groupBy2 function', function () {
+            var params = datasets.groupBy2();
+
+            it('should match types of passing elements', function () {
+                var element1 = params[0] instanceof Array;
+                var element2 = typeof params[1];
+
+                expect(element1).to.eql(true);
+                expect(element2).to.eql('string');
+                expect(params.length).to.eql(2);
+            });
+            it('should contain array of strings and numbers', function () {
+                var array = params[0];
+                var countString = 0;
+                var countInt = 0;
+
+                array.forEach(function (i) {
+                    if (typeof i === 'string') {
+                        countString++;
+                    } else if (typeof i === 'number') {
+                        countInt++;
+                    }
+                });
+                expect(array.length).to.eql(6);
+                expect(countInt).to.eql(3);
+                expect(countString).to.eql(3);
+            });
+            it('should group element of array by length', function () {
+                expect(_.groupBy.apply(_, datasets.groupBy2())).to.eql({
+                    3: ['xyz', 'xyz'],
+                    5: ['combi'],
+                    undefined: [2, 4, 5]
+                });
+            });
+        });
+    });
+
+    describe('includes', function () {
+        describe('incloudes function', function () {
+            var params = datasets.includes();
+            var obj = params[0];
+            var predicate = params[1];
+
+            it('should match types of passing elements', function () {
+                var element1 = params[0] instanceof Object;
+                var element2 = params[1] instanceof Array;
+
+                expect(element1).to.eql(true);
+                expect(element2).to.eql(true);
+                expect(params.length).to.eql(2);
+                expect(params[1].length).to.eql(2);
+            });
+            it('should contain an object with provided properties', function () {
+                expect(Object.keys(obj).length).to.eql(4);
+                expect(_.has(obj, 'firstName')).to.eql(true);
+                expect(_.has(obj, 'lastName')).to.eql(true);
+                expect(_.has(obj, 'users')).to.eql(true);
+                expect(_.has(obj, 'running')).to.eql(true);
+                expect(typeof obj.firstName).to.eql('string');
+                expect(typeof obj.lastName).to.eql('string');
+                expect(typeof obj.users).to.eql('number');
+                expect(typeof obj.running).to.eql('boolean');
+            });
+            it('should include first value in the object', function () {
```

Comment:
What do you mean with first value in the object? I have just choosen some random values inside object to pass the test and I do not know if that was what I was supposed to do.